Completed Cart Functionality

nareshpoonia opened the pull request:
Completed Cart Functionality

nareshpoonia committed:
Completed Cart Functionality

theprogrammerinyou commented on src/components/cart-products.js:
```diff
@@ -0,0 +1,75 @@
+function Cartproducts(props) {
+  const { cartState, cartDispatch } = useCartContext();
+
+  console.log(cartState.cartItem);
```
Do, remove this log while merging.

nareshpoonia replied:
Removed console.log, Thanks for pointing out

theprogrammerinyou reviewed (approved):
Do remove those logs, as part of cleaning code, otherwise code looks good.Thank you.

shivampandey0 commented on src/context/cart-context.js:
```diff
@@ -1,16 +1,63 @@
+    case "REMOVE_FROM_CART":
+      return {
+        ...state,
+        cartArray: state.cartArray.map((item) => {
```
use `Array.filter() `to remove item from list

nareshpoonia replied:
Used Array.filter instead, thanks for suggesting

shivampandey0 reviewed (commented):
Please fix the suggested changes mentioned over call.

nareshpoonia committed:
Destructured props, removed consolelog, removed useReact from functional compo
nents, removed figure, used import in one line if importing multiple items from react, used filter function in cart reducer
